rxjs.nuspec

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="utf-8"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2011/08/nuspec.xsd">
+    <metadata>
+        <id>rxjs</id>
+        <version>5.0.3</version>
```

Comment:
We need to do something to keep this in sync... in fact all of this data... the author email, the version, etc... should be pulled from the package.json when we publish. I think I'd be more in favor of this if it included a script to do so, so I wouldn't have to keep it up to date on each publish.